crates/red_knot_python_semantic/resources/mdtest/comprehensions/basic.md

```diff
@@ -47,6 +47,23 @@ class IntIterable:
 [[reveal_type((x, y)) for x in IntIterable()] for y in IntIterable()]
 ```
 
+## Unpacking in comprehension
```

Comment:
Thanks for adding the test case but I think we should add comprehensive unpacking test cases specific to comprehensions in https://github.com/astral-sh/ruff/blob/e2a38e4c0090cee5c20bde86a8ca9b30bffb2cfa/crates/red_knot_python_semantic/resources/mdtest/unpacking.md as well similar to how it's done for `for` and `with` statement.